.github/workflows/ci.yml

```diff
@@ -21,13 +21,14 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
-      - if: startsWith(matrix.os, 'ubuntu')
+      - if: runner.os == 'Linux'
         run: sudo apt-get install -y build-essential libgl1-mesa-dri libglapi-mesa
                                      libglew-dev libglu1-mesa-dev libosmesa6
                                      libxi-dev mesa-utils pkg-config
-      - run: npm ci
+      - run: rm package-lock.json
+      - run: npm install
```

Comment:
Once `package.json` and `package-lock.json` have been upgraded to `node-gyp==^10.2.0"` then these two lines should be reverted back to `- run: npm ci`.